modules/installation-requirements-user-infra.adoc

```diff
@@ -158,6 +167,7 @@ On your z/VM instance, set up:
 
 * 3 LPARs with 6 IFLs that support SMT2
 * 1 or 2 OSA or RoCE network adapters, or both
+* Hipersockets, which are attached to a node either directly as a device or by bridging with one zVM vSwitch to be transparent to the z/VM guest. To directly connect Hipersockets to a node you must set up a gateway to the external network via a RHEL 8 guest to bridge to the Hipersockets network. See the "Additional resources" section.
```

Comment:
Suggest against the last sentence to see the additional resources section.

Reply:
Also the first sentence uses both `zVM` and `z/VM`. Should they both be the same, or are they different?

Reply:
Yeah, that's the first time I've referred to the Additional resources from main text. In this particular case, the hint to the reader that there's more info on this is really lacking, IMO. But I've removed it to comply with the guidelines.
Yes, should be z/VM, fixed.